berkeley-mobile/Calendar/CalendarViewController.swift

```diff
@@ -214,3 +245,16 @@ extension CalendarViewController {
         upcomingMissingView = MissingDataView(parentView: collectionView, text: "No upcoming events")
     }
 }
+
+extension UIDevice {
+    var hasNotch: Bool {
+        if #available(iOS 11.0, *) {
+            if UIApplication.shared.windows.count == 0 { return false }
+            let top = UIApplication.shared.windows[0].safeAreaInsets.top
```

Comment:
I know this is unlikely to cause any issues, but could you use `.first` instead of `[0]` and safely unwrap it? Just in case, haha.

Reply:
Just noticed you check if `.count == 0` in the line above, so ignore this comment.